3595 terminology changes inline with w3c standards (#328)
KILTprotocol/docs: concepts/05_credentials/01_overview.md

import ThemedImage from '@theme/ThemedImage';

> **Credentials** consist of a set of claims which belong to a **Holder**, are attested by an **Issuer**, and that a **Verifier** can verify.

Reviewer: ```suggestion
**Credentials** consist of a set of claims which belong to a **Holder**, are issued/attested by an **Issuer**, and that a **Verifier** can verify.
```

/>
</center>

To get a credential, a Holder needs to take the following steps:

1. Find a **CType** to base a claim on. Potential Issuers and Verifiers might advertise this information themselves.
2. Make a **claim** containing a set of properties about themselves.

> 3. Fulfil any requirement from your Issuer. For example, accepting their **Terms** and paying a **Quote**.

Reviewer: I think we should get rid of point 2 and change this line as follows:
```suggestion
2. Fulfil any requirement from your Issuer. This may include providing them with **claims** about yourself that you want to see included in a credential, accepting their **Terms**, and paying a **Quote**.
```

> 4. **Request an attestation** from the Issuer.

Reviewer: ```suggestion
4. Request that the Issuer attest to claims about you by issuing a credential.
```

> 5. Wait for the Issuer to **attest** claims.

Reviewer: ```suggestion
5. Wait for the Issuer to **issue** a credential.
```

Once attested, the wrapped claims are considered to be a valid credential.

> To use a Credential, the Holder can generate a Credential-Presentation for a Verifier.

Reviewer: ```suggestion
To use a Credential, the Holder can generate a Verifiable Presentation for a Verifier.
```

The verification would follow this process:

1. The Verifier may request a **Credential** of a CType, along with with properties to reveal.
They also provide a **challenge** to ensure the presentations aren't recycled.
2. The Holder selectively **discloses** the requested properties and signs them along with the challenge to generate a presentation.

> 3. The Verifier **verify** the presentation structure, content and signature, and decides whether they trust the Issuer of the presented credential.

Reviewer: ```suggestion
3. The Verifier **verifies** the presentation structure, content and signature, and decides whether they trust the Issuer of the presented credential.
```
